digits/pretrained_model/tasks/torch_upload.py

```diff
@@ -30,6 +41,30 @@ def get_weights_path(self):
         return os.path.join(self.job_dir, "_Model.t7")
 
     @override
+    def get_deploy_path(self):
+        """
+        Get path to file containing model def for deploy/visualization
+        """
+        return os.path.join(self.job_dir,"original.lua")
```

Comment:
@TimZaman Yeah network.lua would definitely be a better name. Ill try updating this next time I get a chance :)